backend/resume_evaluations/schemas.py

```diff
@@ -0,0 +1,38 @@
+from pydantic import BaseModel
+
+
+class DetailedFeedback(BaseModel):
+    """Individual field-level feedback for each scoring category."""
+
+    skills_match_feedback: str
+    experience_relevance_feedback: str
+    keyword_match_feedback: str
+    projects_feedback: str
+    education_feedback: str
+    formatting_feedback: str
+    additional_value_feedback: str
+
+
+class ResumeEvaluationIn(BaseModel):
+    """Request payload for resume evaluation."""
+
+    resume_text: str
+    job_description: str
+
+
+class ResumeEvaluationOut(BaseModel):
```

Comment:
Follow existing schema conventions.

```suggestion
class ResumeEvaluationResponse(BaseModel):
```